CreeDictionary/tests/CreeDictionary_tests/test_creedictionary_extras.py

```diff
@@ -2,23 +2,14 @@
 # -*- coding: UTF-8 -*-
 
 import pytest
-from CreeDictionary.templatetags.creedictionary_extras import orth
 from django.http import HttpRequest
 from django.template import Context, RequestContext, Template
 from pytest_django.asserts import assertInHTML
 
 
-def test_orth_requires_two_arguments():
-    """
-    orth() original only took one argument, but now it must take two.
-    """
-    with pytest.raises(TypeError):
-        orth("wâpamêw")
-
-
 def test_produces_correct_markup():
     context = Context({"wordform": "wâpamêw"})
```

Comment:
not sure why I missed this test earlier, but thanks for showing me how to test these filters and tags